client/server/render/index.js

```diff
@@ -110,7 +111,7 @@ function render( element, key, req ) {
 
 		logServerEvent( req.context.sectionName, [
 			{
-				name: `ssr.markup_cache.${ renderedLayout ? 'hit' : 'miss' }`,
```

Comment:
`renderedLayout` is always defined here, because this happens after the call to `renderToString` on line 105